### PR TITLE
Add new source: Mangaread

### LINKED ISSUE #425: Mangaread
It would highly appreciated if you could add mangread.org. I came from android and all my mangas are on there. Thanks!

Link: https://mangaread.org/
Language: English

### PR DESCRIPTION
**Pull request by MozkaGit**
## Checklist:
- [x] Updated source's version for individual source changes
- [x] Updated all sources' versions for template changes
- [x] Set appropriate `nsfw` value
- [x] Did not change `id` even if a source's name or language were changed
- [x] Tested the modifications by running it on the simulator or a test device

## Related Issues
- Closes #425 

## Changes:
- Add new source [Mangaread](https://www.mangaread.org/)

## Screenshots:

| Source Version | Browse | In-app Safari |
| :-: | :-: | :-: |
| source_version | source_browse | source_safari |

| Chapters list | Chapter pages | Search |
| :-: | :-: | :-: |
| source_chapters_list | source_chapter | source_search |

https://github.com/Skittyblock/aidoku-community-sources/assets/43102748/e8f26111-97c1-4c60-8619-831d8a53c828

**Thank you in advance for the review.**

### CODE REVIEW

File: src/rust/madara/sources/mangaread/res/filters.json [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+				"id": "yaoi"
+			},
+			{
+				"type": "genre",
+				"name": "Yuri",
+				"id": "yuri"
+			}
+		]
+	}
+]
```
Comment (Skittyblock) [inline src/rust/madara/sources/mangaread/res/filters.json:315]:
```suggestion
]

```

File: src/rust/madara/sources/mangaread/res/source.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+	"info": {
+		"id": "en.mangaread",
+		"lang": "en",
+		"name": "Manga Read",
+		"version": 1,
+		"url": "https://www.mangaread.org/",
```
Comment (Skittyblock) [inline src/rust/madara/sources/mangaread/res/source.json:7]:
```suggestion
		"url": "https://www.mangaread.org",
```

File: src/rust/madara/sources/mangaread/res/source.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+	"info": {
+		"id": "en.mangaread",
+		"lang": "en",
+		"name": "Manga Read",
```
Comment (Skittyblock) [inline src/rust/madara/sources/mangaread/res/source.json:5]: I think Manga Read is used for https://mangaread.co/ in tachiyomi, so we should follow suit and call this MangaRead.org 
```suggestion
		"id": "en.mangareadorg",
		"lang": "en",
		"name": "MangaRead.org",
```

File: src/rust/madara/sources/mangaread/res/filters.json (hunk 1 above)
Comment (Skittyblock) [reply src/rust/madara/sources/mangaread/res/filters.json:315]: json needs a newline

Comment (MozkaGit) [reply src/rust/madara/sources/mangaread/res/filters.json:315]: When I try to commit your suggestion it doesn't worksimage

**Review by Skittyblock — APPROVED** [review]

**Pull request merged**